### PR TITLE
Adds brand guidelines

### PR DESCRIPTION
**Pull request by jonsamp**
# Why

Adding brand guidelines. Since this repo contains our logos meant for others to use, adding the guidelines to the README.md.

# Screenshot

Screenshot 2023-11-22 at 11 43 20 AM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,20 @@
...
 <img width="1553" alt="Screen Shot 2020-01-15 at 12 59 52 PM" src="https://user-images.githubusercontent.com/6455018/72458658-f992f000-3796-11ea-92d9-41159c00dc84.png">
+
+## Brand guidelines
+
+The Expo trademark includes the Expo name and logo, and any word, phrase, image, or other designation that identifies any Expo products. Do not modify the trademarks or use them in an altered way, including suggesting sponsorship or endorsement by Expo, or in a way that confuses Expo with another brand. In other words, the Expo name and logos represent only Expo and should not be used to represent you or your projects, products, or company.
+
+You're encouraged to:
+- Use the permitted Expo name and logos less prominently than your own name and logo to describe that you are using an Expo product. For example: "This app is made using Expo and EAS.".
+- Use a permitted Expo logo to inform others that your project integrates with Expo or EAS. For example: "AwesomeService now works with Expo Router.".
+- Use the permitted Expo name and logos in a blog post, video or news article about Expo.
```
Comment (ide) [inline README.md]: Oxford comma please

**Review by ide — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -3,3 +3,20 @@
...
 
 <img width="1553" alt="Screen Shot 2020-01-15 at 12 59 52 PM" src="https://user-images.githubusercontent.com/6455018/72458658-f992f000-3796-11ea-92d9-41159c00dc84.png">
+
+## Brand guidelines
+
+The Expo trademark includes the Expo name and logo, and any word, phrase, image, or other designation that identifies any Expo products. Do not modify the trademarks or use them in an altered way, including suggesting sponsorship or endorsement by Expo, or in a way that confuses Expo with another brand. In other words, the Expo name and logos represent only Expo and should not be used to represent you or your projects, products, or company.
+
+You're encouraged to:
+- Use the permitted Expo name and logos less prominently than your own name and logo to describe that you are using an Expo product. For example: "This app is made using Expo and EAS.".
+- Use a permitted Expo logo to inform others that your project integrates with Expo or EAS. For example: "AwesomeService now works with Expo Router.".
```
Comment (expjess) [inline README.md]: There are extra periods at the end of these lines (12-13). I tend to solve this by not having punctuation at the end of bullets, but you could also just remove the period that is inside the quotation marks.

**Review by expjess — APPROVED** [review]
Added a note but it's not so critical, can be dealt w later

**Comment by expjess** [comment]
Thank you Jon!!!

**Pull request merged**